### ISSUE TITLE
GPS coordinates not changing

### ISSUE THREAD

**Issue by LTracte** [AnthonyOSX/GO-Bot#181]
This is for v0.0.5.

I've tried changing the coordinates several times, using known good GPS data gathered from Google Map, but it always defaults back to the first place I used this version at.  I wouldnt have known if I hadnt noticed the names of the pokestops that were coming up.  

In a future release, can you also remove all the extra lines of text in the display?  Just leave the relevant data, so it is easier to see whats happening.  

Great job by the way

**Comment by DuIslingr** [comment]
That seems to be Just you. Works fine for me. However I use this to get coordinates of locations. http://www.latlong.net/

**Comment by CTCal** [comment]
Are you using a comma and not a period in the coordinates?

**Comment by LTracte** [comment]
I tried using that latlong.net site, got the same result.  No, I am using a period.  I am copy/pasting coordinates directly from the URL in google maps.  It was working fine on earlier versions, but not v.0.0.5

**Comment by CTCal** [comment]
Sorry, I meant you need to use a comma, for some reason it doesn't change if you leave the period.

**Comment by LTracte** [comment]
so, for example, instead of -73.9739882  i shoudl put -73,9739882?

**Comment by CTCal** [comment]
Yes, exactly, works for me at least. Should probably be fixed though so it accepts either format as it really shouldn't matter.

**Comment by LTracte** [comment]
Tried that, it worked!  Thanks!  It shoudl be decimals, not commas, as that is how coordinates are written.  the comma in a list of coordinates is used to denote lat vs long, not act in place of a decimal.

**Comment by TheReggaeSheep** [comment]
TRY EDITING THE "COORDS" TXT file?
